src/data/text/move_descriptions.h

```diff
@@ -1426,7 +1426,7 @@ static const u8 sGRAVITYDescription[] = _(
     "negating levitation.");
 
 static const u8 sMIRACLE_EYEDescription[] = _(
-    "Makes GHOSTS and evasive\n"
+    "Makes Dark-type and evasive\n"
     "foes easier to hit.");
```

Comment:
```suggestion
    "Negate evasiveness and\n"
    "Dark-type's immunities.");
```
might be a better description.